code/script.js

```diff
@@ -6,14 +6,161 @@ alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+const userName = prompt ("Please enter your name")
+
+console.log(userName)
```

Comment:
Please remove  the console.log() around the code. Generally they need to be deteled when the code has been tested and works as meant to.